python/ray/serve/schema.py

```diff
@@ -544,6 +547,11 @@ class HTTPOptionsSchema(BaseModel, extra=Extra.forbid):
         default=None,
         description="The timeout for HTTP requests. Defaults to no timeout.",
     )
+    timeout_keep_alive_s: int = Field(
```

Comment:
let's name it `keep_alive_timeout_s` (I know it's reverse of uvicorn config, but uvicorn is implementation detail and this matches our other configs such as `request_timeout_s` just above)